.flaskenv

```diff
@@ -0,0 +1,12 @@
+# use publicly acessible env variables in this file
+#   https://flask.palletsprojects.com/en/1.1.x/cli/#environment-variables-from-dotenv
+
+# https://flask.palletsprojects.com/en/1.1.x/cli/#application-discovery
+FLASK_APP=crawler
+
+# https://flask.palletsprojects.com/en/1.1.x/cli/#setting-command-options
+FLASK_RUN_HOST=0.0.0.0
+FLASK_RUN_PORT=8000
+
+# https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features
+FLASK_ENV=development
```

Comment:
Can you move the `SETTINGS_MODULE` env variable to this file too please and remove the `.env` files. It's easier to keep all of our public vars in one place.